src/Apprenticeships/SFA.DAS.Apprenticeships.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -91,23 +93,24 @@ public async Task<ActionResult> GetApprenticeshipStartDate(Guid apprenticeshipKe
     public async Task<ActionResult> CreateApprenticeshipPriceChange(Guid apprenticeshipKey,
         [FromBody] CreateApprenticeshipPriceChangeRequest request)
     {
-        var response = await _apiClient.PostWithResponseCode<PostCreateApprenticeshipPriceChangeApiResponse>(new PostCreateApprenticeshipPriceChangeRequest(
-            apprenticeshipKey,
-            request.Initiator,
-            request.UserId,
-            request.TrainingPrice,
-            request.AssessmentPrice,
-            request.TotalPrice,
-            request.Reason,
-            request.EffectiveFromDate));
+        var response = await _apiClient.PostWithResponseCode<PostCreateApprenticeshipPriceChangeApiResponse>(request.ToApiRequest(apprenticeshipKey));
 
-        if (string.IsNullOrEmpty(response.ErrorContent))
+        if (!string.IsNullOrEmpty(response.ErrorContent))
         {
-            return Ok(new PostCreateApprenticeshipPriceChangeResponse(response.Body.PriceChangeStatus));
+            _logger.LogError("Error attempting to create apprenticeship price change. {statusCode} returned from inner api.", response.StatusCode);
+            return BadRequest();
         }
-               
-        _logger.LogError($"Error attempting to create apprenticeship price change. {response.StatusCode} returned from inner api.", response.StatusCode);
-        return BadRequest();
+
+        var notificationResponse = await _mediator.Send(request.ToNotificationCommand(apprenticeshipKey));
+
+        if (!notificationResponse.Success)
+        {
+            _logger.LogError("Error attempting to send ChangeOfPrice requested notification after successful change of price request");
```

Comment:
As discussed this reads like we are sending a specific notification whereas actually we are passing the notification logic off to the handler to send whatever notifications it deems appropriate.